drivers/wifi/nrf700x/src/wifi_mgmt.c

```diff
@@ -774,6 +819,13 @@ int nrf_wifi_channel(const struct device *dev,
 	}
 
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
+
```

Comment:
i think i have failed to sanitize rpu_ctx_zep here i guess as well. Can you kindly add it?